ssl/os_port.h

```diff
@@ -96,7 +96,8 @@ extern "C" {
 #define be64toh(x) __bswap_constant_64(x)
 #endif
 
-void ax_wdt_feed();
+extern void system_soft_wdt_feed(void);
+#define ax_wdt_feed system_soft_wdt_feed
```

Comment:
Can `ax_wdt_feed` be defined to call `system_soft_wdt_feed` (or another platform-specific function, such as `yield` in Arduino) on the consumer side?